audit/projects/k8s-artifacts-prod/buckets/asia.artifacts.k8s-artifacts-prod.appspot.com/logging.txt

```diff
@@ -1 +1 @@
-gs://asia.artifacts.k8s-artifacts-prod.appspot.com/ has no logging configuration.
+{"logBucket": "k8s-artifacts-gcslogs", "logObjectPrefix": "asia.artifacts.k8s-artifacts-prod.appspot.com"}
```

Comment:
???

Reply:
uh @thockin I think we still need to sort out permissions on this bucket (if we even want to use this project to host this bucket), see https://github.com/kubernetes/k8s.io/issues/904#issuecomment-816431175

Reply:
Will track this under https://github.com/kubernetes/k8s.io/issues/904#issuecomment-832195772 as something to followup on